Check tests that sometimes fails

At the commit hash https://github.com/babylonchain/babylon/commit/0668b47229ae5da83e4c720c6dfe6926486896e1

The CI pointed a test [failure](https://app.circleci.com/pipelines/github/babylonchain/babylon/2316/workflows/b2363d7a-6b1e-4ead-bb71-cd8c5cf66563/jobs/4621) 

But running it again it [passed](https://app.circleci.com/pipelines/github/babylonchain/babylon/2316/workflows/340ffa39-d749-45fa-ae46-50ccb9f72852) 

Verify why sometimes it fails, sometimes it passes 

```shell
--- FAIL: FuzzFinalityProviderPowerAtHeight (0.02s)
    --- FAIL: FuzzFinalityProviderPowerAtHeight/seed#7 (0.00s)
        store.go:206: <nil> DBG loadVersion ver=0
        store.go:244: <nil> DBG loadVersion commitID hash= key="KVStoreKey{0xc000debc00, btcstaking}" ver=0
        store.go:62: <nil> INF Upgrading IAVL storage for faster queries + execution on live state. This may take a while commit=436F6D6D697449447B5B5D3A307D store_key="KVStoreKey{0xc000debc00, btcstaking}" version=0
        store.go:76: <nil> DBG Finished loading IAVL tree
        grpc_query_test.go:305: 
                Error Trace:    /home/circleci/project/x/btcstaking/keeper/grpc_query_test.go:305
                                                        /usr/local/go/src/reflect/value.go:596
                                                        /usr/local/go/src/reflect/value.go:380
                                                        /usr/local/go/src/testing/fuzz.go:335
                Error:          Target error should be in err chain:
                                expected: "voting power table has not been updated"
                                in chain: 
                Test:           FuzzFinalityProviderPowerAtHeight/seed#7
```

--- a/x/btcstaking/keeper/grpc_query_test.go
+++ b/x/btcstaking/keeper/grpc_query_test.go
@@ -296,7 +296,7 @@ func FuzzFinalityProviderPowerAtHeight(f *testing.F) {
 
 		// case where the voting power store is not updated in
 		// the given height
-		requestHeight := datagen.RandomIntOtherThan(r, int(randomHeight), 100) + 1
+		requestHeight := randomHeight + datagen.RandomInt(r, 10) + 1
 		req2 := &types.QueryFinalityProviderPowerAtHeightRequest{
 			FpBtcPkHex: fp.BtcPk.MarshalHex(),
 			Height:     requestHeight,

